### PR TITLE
UE-1101: Add data/mine endpoint.

### PR DESCRIPTION
**Pull request by nimit-ddprop**

### CODE REVIEW

File: src/bootstrap.py [hunk 1]
```diff
@@ -1,25 +1,99 @@
...
+    if server:
+        create_server()
+
     __INITED = True
+    return
+
 
+def append_sys_path() -> None:
+    """Append path to sys module.
+    By do this we can use any modules inside project.
```
Comment (abrarazeem) [inline src/bootstrap.py]: correct statement should be By "doing"

File: src/wsgi.py [hunk 2]
```diff
@@ -1,32 +1,34 @@
...
 
 app.register_blueprint(recommendation)
+app.register_blueprint(data_blueprint)
 
 
 @app.route("/")
-def echo():
+def echo() -> Tuple[Response, int]:
+    """Echo
+    Echo API only. It's do one thing.
```
Comment (abrarazeem) [inline src/wsgi.py]: it's doing one thing.

File: src/bootstrap.py (hunk 1 above)
Comment (ethaizone) [reply src/bootstrap.py]: Yes. I miss that.

File: src/wsgi.py (hunk 2 above)
Comment (ethaizone) [reply src/wsgi.py]: Ok. Maybe I change to "It does one thing.".

File: src/bootstrap.py (hunk 1 above)
Comment (nimit-ddprop) [reply src/bootstrap.py]: Fixed. (Yesterday I used private account. Why I can access it?)

File: src/wsgi.py (hunk 2 above)
Comment (nimit-ddprop) [reply src/wsgi.py]: Fixed. I update to another message.

**Pull request merged**